readme.md

```diff
@@ -164,6 +164,15 @@ $ antibody bundle mafredri/zsh-async
 $ antibody bundle sindresorhus/pure
 ```
 
+### [zplug](https://github.com/zplug/zplug)
+
+Update your `.zshrc` file with the following lines (order matters):
+
+```console
+$ zplug mafredri/zsh-async, from:github, defer:0  # Load this first
+$ zplug sindresorhus/pure, use:pure.zsh, from:github, as:theme
```

Comment:
@sindresorhus not really relevant for this PR, but in my opinion the `console` and `$` don't really make sense here since they represent lines in a file (`.zshrc`).

Thoughts? If you agree I can submit a PR to change this for all the examples where it doesn't make sense.